src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,319 @@
+        /// <param name="delay">The delay between each measurements. Note the unit does not directly to hz, a value of 20 maps to about 100 hz.</param>
+        public void SetMeasurementRepetitionMode(MeasurementRepetitionMode measurementRepetitionMode, int? count = null, int? delay = null)
+        {
+            if (count.HasValue && (count < 2 || count > 254))
+            {
+                throw new ArgumentOutOfRangeException("Count must be between 2 and 254");
+            };
+
+            switch (measurementRepetitionMode)
+            {
+                case MeasurementRepetitionMode.Repeat:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, (byte)count);
+                    break;
+                case MeasurementRepetitionMode.RepeatIndefinitely:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0xff);
+                    break;
+                case MeasurementRepetitionMode.Off:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0x00);
+                    break;
+            }
+
+            if (delay != null)
+            {
+                WriteRegister(Register.MEASURE_DELAY, (byte)delay.Value);
+
+                // Set mode to use custom delay.
+                var currentAcqMode = AcquistionMode;
+                currentAcqMode |= AcquistionModeFlag.UseDefaultDelay;
+                AcquistionMode = currentAcqMode;
+            } else
+            {
+                // Set mode to use default delay.
+                var currentAcqMode = AcquistionMode;
+                currentAcqMode &= ~AcquistionModeFlag.UseDefaultDelay;
+                AcquistionMode = currentAcqMode;
+            }
+
+            // Kick it off with a single acquire command.
+            WriteRegister(Register.ACQ_COMMAND, 0x04);
+        }
+
+        /// <summary>
+        /// Set a new I2C address.
+        /// </summary>
+        /// <param name="address">new address, valid values are 7-bit values with 0 in the LSB.</param>
+        public void SetI2CAddress(byte address)
+        {
+            if ((address & 1) == 1)
+            {
+                throw new ArgumentOutOfRangeException("Address must have 0-bit in the LSB.");
+            }
+
+            // Read in the unit's serial number.
+            Span<byte> rawData = stackalloc byte[2] { 0, 0 };
+            ReadBytes(Register.UNIT_ID, rawData);
+
+            // Write serial number to I2C_ID.
+            WriteRegister(Register.I2C_ID_HIGH, rawData[1]);
+            WriteRegister(Register.I2C_ID_LOW, rawData[0]);
+
+            // Write the new address.
+            WriteRegister(Register.I2C_SEC_ADDR, address);
+
+            // Disable the default address
+            WriteRegister(Register.I2C_CONFIG, 0x08);
+        }
+
+        #region Device Registers
+
+        /// <summary>
+        /// Get the distance measurement in cm.
+        /// </summary>
+        public int Distance {
```

Comment:
1. Braces on new line (here and everywhere)
2. Since this is not doing the measurement and just reads back the last measurement (if I understand correctly) this should be called LastDistance to make it clear.

Reply:
Will fix.